docs/book/v5/core-features/account-anonymization.md

```diff
@@ -0,0 +1,32 @@
+# Account anonymization
+
+According to the GDPR, companies that record personal data from EU citizens must delete said data if its owner requests its deletion.
+An alternative is to anonymize the data, according to [this article](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/dealing-citizens/do-we-always-have-delete-personal-data-if-person-asks_en).
+
+## What is 'personal data'?
+
+According to [this article](https://commission.europa.eu/law/law-topic/data-protection/reform/what-personal-data_en), personal data is:
+
+- A name and surname.
+- A home address.
+- An email address such as name.surname@company.com.
+- An identification card number.
+- Location data (for example the location data function on a mobile phone).
+- An Internet Protocol (IP) address.
+- A cookie ID.
+- The advertising identifier of your phone.
+- Data held by a hospital or doctor, which could be a symbol that uniquely identifies a person.
+
+Out of the box, Dotkernel Frontend saves the user's name (firstname and lastname) and email (identity).
```

Comment:
Maybe we should also mention the avatar - in case that someone uses an image with their face in it.
Now that I think, I think that we should also delete the user avatar on user account anonymization.

Reply:
i create an issue for this in frontend and will expand the docs too
https://github.com/dotkernel/frontend/issues/482